src/Tools/dotnet-monitor/ConfigurationKeys.cs

```diff
@@ -19,5 +19,7 @@ internal static class ConfigurationKeys
         public const string Storage = nameof(Storage);
 
         public const string DefaultProcess = nameof(DefaultProcess);
+
+        public const string Logging = nameof(Logging);
```

Comment:
Note that some already existing configuration sections have named constants. For example https://docs.microsoft.com/en-us/dotnet/api/microsoft.aspnetcore.hosting.webhostdefaults?view=aspnetcore-5.0. I don't know if Logging does, but it may be worthwhile to check.